README.md

```diff
@@ -1,31 +1,159 @@
 <p align="center">
-  <img src="https://raw.githubusercontent.com/amazeeio/lagoon/master/docs/images/lagoon-logo.png" alt="The Lagoon logo is a blue hexagon split in two pieces with an L-shaped cut" width="40%">
+  <img src="./Lagoon_OG.png" alt="The Lagoon logo is a blue hexagon split in two pieces with an L-shaped cut" width="40%">
 </p>
 
-# amazee.io Lagoon - Docker Build and Deploy System for OpenShift & Kubernetes
+# Lagoon - the developer-focused application delivery platform
 
 Lagoon solves what developers are dreaming about: A system that allows developers to locally develop their code and their services with Docker and run the exact same system in production. The same Docker images, the same service configurations and the same code.
 
-Please reference our [documentation](https://lagoon.readthedocs.io/) for detailed information on using, developing, and administering Lagoon.
+> As an application delivery **platform**, the primary focus of Lagoon is as a tool for the deployment, management, security and operation of multiple applications, utilising cloud-native technologies, but with minimal imposition or knowledge expectation on the developers of those applications.
+
+Lagoon is fully open-source, built on open-source tools, built collaboratively with our users.
+
+## Installing Lagoon
+
+*Note that is not necessary to install Lagoon on to your local machine if you are looking to maintain websites hosted on Lagoon.*
+
+Lagoon can be installed:
+
+- Locally (for evaluation, testing, debugging or development) using [Helm](https://helm.sh/) charts and [kind](https://kind.sigs.k8s.io/)
+- Into your managed Kubernetes cloud provider of choice - it's running in production on [Amazon Elastic Kubernetes Service](https://aws.amazon.com/eks/), [Azure Kubernetes Service](https://azure.microsoft.com/en-au/services/kubernetes-service/), and [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/), but we are confident that it will also run on any other managed Kubernetes service.
+
+For more information on developing or contributing to Lagoon, head to https://docs.lagoon.sh/lagoon/contributing-to-lagoon
+
+For more information on installing and administering Lagoon, head to https://docs.lagoon.sh/lagoon/administering-lagoon
+
+## Lagoon architecture
+
+Lagoon is comprised of two main components, Lagoon Core and Lagoon Remote, and a number of other third-party services, Operators and Controllers.  In a full production setting, it is envisaged that Lagoon Core and Remote are installed into different Kubernetes Clusters (one Core can serve many Remotes), but they can also be installed into the same cluster if required.  Additionally, in advanced situations, a Lagoon Remote can contain multiple build-deploy controllers, connected to different Lagoon Cores (but this is only really recommended for testing purposes).
```

Comment:
I would not mention the Additionally piece, it's super super edgy and makes the whole thing here more complex to understand